corehq/apps/app_manager/templates/app_manager/partials/filter_configs.html

```diff
@@ -17,37 +17,51 @@
-                    <input type="text"
-                           class="choice_filter"
-                           style="width:300px"
-                           data-bind="value: filter.selectedValue.value"/>
-                    <input type="hidden" data-bind="value: dynamicFilterName">
-                </div>
-                <div data-bind="visible: filter.selectedValue.doc_type() === 'StaticChoiceFilter'">
-                    <select data-bind="
-                        options: filter.choices,
-                        optionsText: 'display',
-                        optionsValue: 'value',
-                        value: filter.selectedValue.select_value
-                    "></select>
+                <div style="padding-top:10px">
+                    <div data-bind="visible: filter.selectedValue.doc_type() === 'AutoFilter'">
+                        <select data-bind="
+                            options: report.filterConfig.autoFilterTypes,
+                            value: filter.selectedValue.filter_type
+                        "></select>
+                    </div>
+                    <div data-bind="visible: filter.selectedValue.doc_type() === 'StaticDatespanFilter'">
+                        <select data-bind="
+                            options: report.filterConfig.date_range_options,
+                            value: filter.selectedValue.date_range
+                        "></select>
+                    </div>
+                    <div data-bind="visible: filter.selectedValue.doc_type() === 'CustomDatespanFilter'">
+                        <span>today() - </span><span data-bind="text: filter.display"></span>
+                        <select style="width:85px;"
+                                data-bind="
+                                    options: report.filterConfig.date_operators,
+                                    value: filter.selectedValue.operator
+                        "></select>
+                        <input type="number" style="width:45px;" data-bind="value: filter.selectedValue.date_number"/>
+                        <span data-bind="visible: filter.selectedValue.operator() === 'between'">and</span>
+                        <input type="number" style="width:45px;" data-bind="
+                            value: filter.selectedValue.date_number2,
+                            visible: filter.selectedValue.operator() === 'between'
+                        "/>
+                    </div>
+                    <div data-bind="visible: filter.selectedValue.doc_type() === 'CustomDataAutoFilter'">
+                        <input type="text" data-bind="value: filter.selectedValue.custom_data_property"/>
+                    </div>
+                    <div data-bind="visible: filter.selectedValue.doc_type() === 'StaticChoiceListFilter'">
+                        <input type="text"
+                               class="choice_filter"
+                               style="width:300px"
```

Comment:
is there a reason this is extra wide? i know it was already this big & you just moved it, just wondering.